sources/slack_pipeline.py

```diff
@@ -45,8 +41,25 @@ def select_resource(selected_channels: List[str]) -> None:
         selected_channels=selected_channels,
         start_date=datetime(2023, 9, 1),
         end_date=datetime(2023, 9, 8),
+    ).with_resources("channels", "1-announcements", "dlt-github-ci")
+
+    load_info = pipeline.run(
+        source,
+    )
+    print(load_info)
+
+
+def get_users() -> None:
+    """Execute a pipeline that will load Slack users list."""
+
+    pipeline = dlt.pipeline(
+        pipeline_name="slack", destination="duckdb", dataset_name="slack_data"
     )
 
+    source = slack_source(
```

Comment:
cool! but please remember to add it to tests!

Reply:
I am just waiting for the permissions in the main channel account to create the tests.